lib/generators/rolemodel/source_map/source_map_generator.rb

```diff
@@ -0,0 +1,11 @@
+#frozen_string_literal: true
+
+module Rolemodel
+  class ReadmeGenerator < Rails::Generators::Base
+    source_root File.expand_path(__dir__)
+
+    def copy_source_map_files
+      copy_entry 'source_map', 'app'
```

Comment:
Should this also be copying the actual source map middleware, not just the initializer?

Reply:
Also, the js files don't seem to be getting copied in either.